pyface/tests/test_about_dialog.py

```diff
@@ -8,16 +8,26 @@
 from ..toolkit import toolkit_object
 from ..window import Window
 
+GuiTestAssistant = toolkit_object('util.gui_test_assistant:GuiTestAssistant')
+no_gui_test_assistant = (GuiTestAssistant.__name__ == 'Unimplemented')
+
 ModalDialogTester = toolkit_object('util.modal_dialog_tester:ModalDialogTester')
 no_modal_dialog_tester = (ModalDialogTester.__name__ == 'Unimplemented')
 
 
-class TestAboutDialog(unittest.TestCase):
+@unittest.skipIf(no_gui_test_assistant, 'No GuiTestAssistant')
+class TestAboutDialog(unittest.TestCase, GuiTestAssistant):
 
     def setUp(self):
-        self.gui = GUI()
+        GuiTestAssistant.setUp(self)
```

Comment:
I prefer to do explicit call here, rather than super, because I found it surprising that `TestCase` _didn't_ implement a trivial `setUp` and `tearDown`, so the explicit call flags the future readers of the code that something not usual is happening here.